plugins/woocommerce/includes/class-wc-cart-totals.php

```diff
@@ -137,8 +137,12 @@ public function __construct( &$cart = null ) {
 			throw new Exception( 'A valid WC_Cart object is required' );
 		}
 
+		// Check if customer is VAT exempt, if customer is defined.
+		$customer               = $cart->get_customer();
+		$is_customer_vat_exempt = $customer && $customer->get_is_vat_exempt();
+
```

Comment:
This ensures we have a customer object before calling `get_is_vat_exempt`. Prevents possible fatal errors if the customer is null. This shouldn't happen in regular execution but was discovered on one test site.